modules/prometheus/default.nix

```diff
@@ -35,9 +40,18 @@ in
     };
   };
 
+  networking.firewall.extraInputRules = ''
+    ip6 saddr $prometheus_inet6 tcp dport 9300
```

Comment:
If we cannot interpolate the port number, I suggest we preserve the `# prometheus-nixos-exporter` comment from before.